Sources/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -280,6 +280,23 @@ public extension Array {
             value += slice
         }
     }
+    
+    
+    /// SwifterSwift: Returns an array of slices of lenght "size" from the array.  If array can't be split evenly, the final slice will be the remaining elements.
+    ///
+    /// - Parameters:
+    ///   - size: The size of the slices to be returned.
+    public func split(size: Int) -> [ArraySlice<Element>]? {
```

Comment:
@SD10 Hey man :)
About the memory implications, I think it's the kind of extension that the people who use knows the concerns about ArraySlice. But i don't know, may it's something to consider. 
I just return the ArraySlice because I wanted to follow the same, pattern of the native mehtods that do similar stuff like `split(whereSeparator: (Element) throws -> Bool)-> [ArraySlice<Element>]` and by the way the name split is also to follow the pattern of this methods.
I thought about `slices(ofSize:)` or `split(ofSize:)`, let me know what you think?